README.md

```diff
@@ -1,3 +1,12 @@
+# Before Getting Started
+
+Please be sure when running the front end repository that you have the correct versions of npm and node.
+The correct version can be found and installed [here](https://nodejs.org/download/release/v14.17.0/).
```

Comment:
Node 14 is no longer supported as of April 30. We should upgrade.

Reply:
I will test out the latest release and update the readme after confirming that everything is building and running well.

Reply:
Updated readme.